cfme/tests/automate/test_method.py

```diff
@@ -650,3 +652,88 @@ def test_method_for_log_and_notify(request, klass, notify_level, log_level):
             result.validate(wait="60s")
     else:
         result.validate(wait="60s")
+
+
+@pytest.mark.tier(1)
+@pytest.mark.meta(blockers=[BZ(1698184, forced_streams=['5.10'])], automates=[1698184])
+def test_null_coalescing_fields(request, klass):
+    """
+    Bugzilla:
+        1698184
+
+    Polarion:
+        assignee: ghubale
+        initialEstimate: 1/8h
+        caseimportance: high
+        caseposneg: positive
+        testtype: functional
+        startsin: 5.10
+        casecomponent: Automate
+        tags: automate
+        testSteps:
+            1.  Create a Ruby method or Ansible playbook method with Input Parameters.
+            2.  Use Data Type null coalescing
+            3.  Make the default value something like this : ${#var3} || ${#var2} || ${#var1}
+        expectedResults:
+            1.
+            2.
+            3. Normal null coalescing behavior
+    """
+    var1, var2, var3, var4 = [fauxfactory.gen_alpha() for _ in range(4)]
+
+    klass.schema.add_fields(
+        *[
+            {"name": var, "type": "Attribute", "data_type": "String", "default_value": value}
+            for var, value in [[var1, "fred"], [var2, "barney"], [var3, " "]]
```

Comment:
**Optional**: With `var1` set to _fred_, I would love it if you could name `var2` _george_ :grinning: